components/tabs/tabs.js

```diff
@@ -301,6 +307,8 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(RtlMixin(FocusVisiblePolyf
 			}
 		};
 
+		if (this._tabInfos.length > 0) this._tabInfos[0].activeFocusable = true;
```

Comment:
I think this is already being handled in the slot change handler below, ya?

Reply:
Yep, you're right. I forgot that we'll always get a `slotchange` event firing on first update.